### ISSUE TITLE
Chips usage of emojis on chip_icon_only

### ISSUE THREAD

**Issue by rphlwnk** [UI-Lovelace-Minimalist/UI#23]
I'm struggling on using icons with the **chips_icon_only** template.
I was trying to use emojis, turned out it only works with hass: or mdi: icons

![image](https://user-images.githubusercontent.com/34039879/140644359-5061496b-ae30-4e57-876d-b3171e1278ac.png)

```
- type: 'custom:button-card'
  template: chip_icon_only
  variables:
    ulm_chip_icon_only: '❤️'
- type: 'custom:button-card' 
  template: chip_icon_only
  variables:
    ulm_chip_icon_only: 'mdi:car'
```
Maybe change or add another one that can handle emojis? Or do i miss something?

![image](https://user-images.githubusercontent.com/34039879/140644660-8bd38964-5bb6-47d1-9edb-31ddb5178e3c.png)

```
- type: 'custom:button-card'
  template: chip_emoji_only
  variables:
    ulm_chip_icon_only: '❤️'

chip_emoji_only:
  template: chips
  variables:
    icon: '❔'
  show_icon: true
  label: |
    [[[ return variables.ulm_chip_icon_only ? variables.ulm_chip_icon_only : variables.icon ]]]
```

**Comment by Paddy0174** [comment]
Will be fixed with #27 thanks to `@CM000n`

**Comment by Paddy0174** [comment]
Closed with commit bd6cb848dbbbf37911f721f140daa8b9dd6013ea
